.gitignore

```diff
@@ -43,39 +40,49 @@ doxygen/
 /keyboards/*/*/*/*/*/keymaps/Makefile
 
 # Eclipse/PyCharm/Other IDE Settings
+*.iml
+.browse.VC.db*
 .cproject
+.idea
+.idea/
 .project
 .settings/
-.idea
-*.iml
-.browse.VC.db*
-*.stackdump
+.vagrant/
+
+# ?
+.dep
+.history/
+build/
+cmake-build-debug
+CMakeLists.txt
+tags
```

Comment:
Do we need these? QMK doesn't use CMake, and I don't know what the others are for.

Reply:
`tags` is used by ctags/etags.